build.gradle

```diff
@@ -1004,7 +1004,7 @@ project('test:system') {
         environment "zookeeperOperatorVersion",System.getProperty("zookeeperOperatorVersion", "latest")
         // The below properties are used to to specify the pravega published chart and repository to deploy pravega,bookkeeper & zookkeeper operators using helm docker image properties
         environment "publishedChartName",System.getProperty("publishedChartName", "pravega")
-        environment "helmRepository",System.getProperty("helmRepository", "pravega")
+        environment "helmRepository",System.getProperty("helmRepository", "https://charts.pravega.io")
```

Comment:
`pravega` is not a valid default..

Reply:
Nit: looks like there is missing a space after comma in that new line (and in previous ones).